src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/SymmetricKeyWrap.cs

```diff
@@ -30,10 +30,14 @@ internal static byte[] TripleDESKeyWrapEncrypt(byte[] rgbKey, byte[] rgbWrappedK
 
             // generate a random IV
             byte[] rgbIV = new byte[8];
+#if NETCOREAPP || NETSTANDARD2_1
```

Comment:
This library doesn't build for either of these configurations. There's no benefit to adding this.